traffic_ops/traffic_ops_golang/config/config.go

```diff
@@ -99,8 +100,9 @@ type ConfigTrafficOpsGolang struct {
 	RiakPort                 *uint                      `json:"riak_port"`
 	WhitelistedOAuthUrls     []string                   `json:"whitelisted_oauth_urls"`
 	OAuthClientSecret        string                     `json:"oauth_client_secret"`
-	RoutingBlacklist         `json:"routing_blacklist"`
-	SupportedDSMetrics       []string `json:"supported_ds_metrics"`
+	RoutingBlacklist                                    `json:"routing_blacklist"`
+	SupportedDSMetrics       []string                   `json:"supported_ds_metrics"`
+	TLSConfig                *tls.Config                `json:"tls_config"`
```

Comment:
This file needs to be `go fmt`-ed

Reply:
Uh, I don't know how to do that. Is that going to keep this PR from being accepted?

Reply:
> I don't know how to do that.

To `go fmt` a file you just do `go fmt {{file}}`.

> Is that going to keep this PR from being accepted?

Well, yeah, it's listed in [the docs for Traffic Ops Go development requirements](https://traffic-control-cdn.readthedocs.io/en/latest/development/traffic_ops.html#go-implementation-requirements), but it's also an extremely simple fix.

> ... I was following the format that was already there.

Well, you lined up the struct tags. But you can see that [you actually changed the format](https://github.com/apache/trafficcontrol/pull/4933/files#diff-8f0e586303d36096ee3c6ae7a0a7bffdL102-R105):
```diff
-  RoutingBlacklist         `json:"routing_blacklist"`
+  RoutingBlacklist                                    `json:"routing_blacklist"`
```